test/ipc_os_prov_consumer.c

```diff
@@ -14,8 +14,8 @@
 #include "ipc_os_prov_common.h"
 
 int main(int argc, char *argv[]) {
-    if (argc < 2) {
-        fprintf(stderr, "usage: %s <port> [shm_name]\n", argv[0]);
```

Comment:
IMHO it should stay as it was. The consumer can use also another SHM file or none.